mulighetsrommet-arena-adapter/src/test/kotlin/no/nav/mulighetsrommet/arena/adapter/consumers/TiltakgjennomforingEndretConsumerTest.kt

```diff
@@ -53,7 +54,10 @@ class TiltakgjennomforingEndretConsumerTest : FunSpec({
                 )
             )
 
-            val consumer = createConsumer(database.db, MockEngine { respondOk() })
+            val consumer = createConsumer(
```

Comment:
Ser ut som at det blir ganske mye diff, som egentlig ikke hadde trengt å være en diff her (bare en observasjon). Har du egne innstillinger for formattering?

Reply:
Bruker klint, men ser ut som min er satt opp til å ha litt kortere linjer kanskje? Jeg bruker bare default formateringer, men vi har kanskje en egen for prosjektet?

Reply:
Enig :) Det er vel strengt tatt satt opp for prosjektet allerede i `build.gradle` og det er bare en regel som avviker fra standardinnstillinger til ktlint, så litt rart at det blir annerledes hos deg. Det er mulig intellij-pluginen opererer med andre defaults en gradle-pluginen :thinking: